src/Storage/Storage.Management/help/Enable-AzStorageBlobDeleteRetentionPolicy.md

```diff
@@ -52,7 +52,7 @@ This command enables delete retention policy for the Blob service, and set delet
 ## PARAMETERS
 
 ### -AllowPermanentDelete
-Allow deletion of the soft deleted blob versions and snapshots.
+Allow deletion of the soft deleted blob versions and snapshots. Please review the conditions applicable for [Permanent Delete](https://learn.microsoft.com/en-us/rest/api/storageservices/set-blob-service-properties?tabs=azure-ad#:~:text=RetentionPolicy/AllowPermanentDelete).
```

Comment:
Please also update https://github.com/Azure/azure-powershell/blob/93a3a9f425ea91f22a409617fa17f24c7bb9e8e6/src/Storage/Storage.Management/Blob/EnableAzureStorageBlobDeleteRetentionPolicy.cs#L90.
They should be aligned.